src/utils/proto-methods.js

```diff
@@ -0,0 +1,2 @@
+export var arraySlice     = Array.prototype.slice;
```

Comment:
We need a comment that describe why we need this stuff as well, with the link to the GH issue.